test/kuttl/testmetrics/metrics-provider-dummy/00-install.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: podtatometric
+  namespace: keptn-system
+spec:
+  provider:
+    name: "dummy"
+  query: "random"
+  fetchIntervalSeconds: 5
+
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetricsProvider
+metadata:
+  name: dummy
+  namespace: keptn-system
+spec:
+  type: dummy
+  targetServer: "http://www.randomnumberapi.com/api/v1.0/"  # string
```

Comment:
not sure this will work since you cannot assert what value will be returned everytime

Reply:
I don't think I am returning any value just the test from the return function.